src/modules/musig/musig.md

```diff
@@ -0,0 +1,196 @@
+MuSig - Rogue-Key-Resistant Multisignatures Module
+===========================
+
+This module implements the MuSig [1] multisignature scheme. The majority of
+the module is an API designed to be used by signing or auditing participants
+in a multisignature scheme. This involves a somewhat complex state machine
+and significant effort has been taken to prevent accidental misuse of the
+API in ways that could lead to accidental signatures or loss of key material.
```

Comment:
It should be mentioned that MuSig creates a Schnorr signature (compatibe with the Schnorr BIP) here or somewhere else.